pkgs/development/python-modules/wandb/default.nix

```diff
@@ -141,6 +136,10 @@ buildPythonPackage rec {
 
   pythonRelaxDeps = [ "protobuf" ];
 
+  pytestFlagsArray = [
+    "tests/pytest_tests"
+  ];
```

Comment:
can we add a comment explaining why this is necessary?

Reply:
How about this?
https://github.com/NixOS/nixpkgs/compare/f09e41ef28250fa2ade4cfb0bcf71e173fc6c91d..5f4ded8f19d946061812603f13cee9e18be5fe62

Reply:
sure that's fine by me